roles/nginx/defaults/main.yml

```diff
@@ -16,3 +16,8 @@ nginx_cache_path: /var/cache/nginx
 nginx_cache_key_storage_size: 10m
 nginx_cache_size: 250m
 nginx_cache_inactive: 1h
+
+
+# This should be turned on if you are using `gzip_static_assets_on_deploy`.
+# This value should be qouted.
```

Comment:
Separating `gzip_static_assets_on_deploy` and `nginx_gzip_static`:
- enabling `gzip_static` globally gives performance penalty
- allow developers to customize nginx config

See: https://github.com/roots/trellis/blob/c8f91bf808f1e5b4754c15e3d19ba5079ca0beb9/roles/nginx/templates/nginx.conf.j2#L165-L169